release/common_test_cases.hh

```diff
@@ -50,5 +50,7 @@ TEST(CommonSingleFile, IncludesMathFunctions) {
     EXPECT_DOUBLE_EQ(sin(radians(get_value<double>(PI / mag<2>()))), 1.0);
 }
 
+TEST(CommonSingleFile, MixedSignQuantityComparisonWorks) { EXPECT_LT(meters(-1), meters(1u)); }
```

Comment:
Huh, I totally missed this file when doing the Matcher conversions, didn't realize there was code outside of `au/`.

Reply:
Oops... I thought I had a good `EXPECT_THAT` habit going now.  Oh well, we can convert the whole file at once anyway.